pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -172,118 +174,123 @@ type CertificateRequestPolicyAllowedX509Subject struct {
 	// for.
-	// Values are inclusive (i.e. a min value of `1h` will accept a duration of
+	// Values are inclusive (i.e. a value of `1h` will accept a duration of
 	// `1h`). MinDuration and MaxDuration may be the same value.
-	// An omitted field or value of `nil` permits any minimum duration.
-	// If MinDuration is defined, a duration _must_ be requested on the
-	// CertificateRequest.
+	// If set, a duration _must_ be requested on the CertificateRequest.
+	// An omitted field implies no minimum constraint on duration.
 	// +optional
 	MinDuration *metav1.Duration `json:"minDuration,omitempty"`
 
 	// MaxDuration defines the maximum duration a certificate may be requested
 	// for.
-	// Values are inclusive (i.e. a max value of `1h` will accept a duration of
-	// `1h`). MaxDuration and MinDuration may be the same value.
-	// An omitted field or value of `nil` permits any maximum duration.
-	// If MaxDuration is defined, a duration _must_ be requested on the
-	// CertificateRequest.
+	// Values are inclusive (i.e. a value of `1h` will accept a duration of
+	// `1h`). MinDuration and MaxDuration may be the same value.
+	// If set, a duration _must_ be requested on the CertificateRequest.
+	// An omitted field implies no maximum constraint on duration.
 	// +optional
 	MaxDuration *metav1.Duration `json:"maxDuration,omitempty"`
 
-	// PrivateKey defines the shape of permissible private keys that may be used
-	// for the request with this policy.
-	// An omitted field or value of `nil` permits the use of any private key by
-	// the requestor.
+	// PrivateKey defines constraints on what shape of private key a
+	// CertificateRequest is allowed to use for its request.
+	// An omitted field implies no private key shape constraints.
 	// +optional
 	PrivateKey *CertificateRequestPolicyConstraintsPrivateKey `json:"privateKey,omitempty"`
 }
 
 // CertificateRequestPolicyConstraintsPrivateKey defines constraints on what
-// shape of private key is permissible for a CertificateRequest to have used
-// for its request.
+// shape of private key a CertificateRequest is allowed to use for its request.
 type CertificateRequestPolicyConstraintsPrivateKey struct {
 	// Algorithm defines the allowed crypto algorithm that is used by the
 	// requestor for their private key in their request.
-	// An omitted field or value of `nil` permits any Algorithm.
+	// An omitted field permits any Algorithm.
 	// +optional
 	Algorithm *cmapi.PrivateKeyAlgorithm `json:"algorithm,omitempty"`
 
-	// MinSize defines the minimum key size a requestor may use for their private
-	// key.
+	// MinSize defines the minimum key size a requestor may use for their
+	// private key.
 	// Values are inclusive (i.e. a min value of `2048` will accept a size
 	// of `2048`). MinSize and MaxSize may be the same value.
-	// An omitted field or value of `nil` permits any minimum size.
+	// An omitted field permits any minimum size.
```

Comment:
I can't keep track of whether multiple policies can apply concurrently w/ a requirement for all to pass... Assuming it's an all must pass, this wording change makes sense:

```suggestion
	// An omitted field implies no additional minimum constraint on size.
```

Reply:
Ref. https://cert-manager.io/docs/policy/approval/approver-policy/#configuration, I don't think the suggested rewording is correct:
![image](https://github.com/cert-manager/approver-policy/assets/1142578/2e4e20db-b473-4f38-8746-425a012943e5)

Reply:
ok, so it isn't an _all_ (**and**), it's an _any_ (**or**) matching policy.

I think the text is still wrong, although I don't know how to fix it.

This is the `CertificateRequestPolicy`, right? And it's defining parameters for a `CertificateRequest`, right?
But the `CertificateRequest` will not have a `minimum size` field, it'll have a `size` field, which will just not have any particular minimum size, right?

```suggestion
	// An omitted field allows size to be as small as possible.
```

Reply:
good enough

Reply:
Probably should be `applies` for consistency w/ others